### ISSUE TITLE
Migrate to GlobalFishingWatch GitHub org

### ISSUE THREAD

**Issue by geowurster** [GlobalFishingWatch/pelagos-data#4]
_For https://github.com/SkyTruth/GFW-Tasks/issues/26_

@redhog @pwoods25443 @enriquetuya @andres-arana Anyone know if this repo is still being used for anything?  If not we should transfer ownership.

**Comment by enriquetuya** [comment]
@geowurster I tried finding mentions to this repo on slack. The only ones I found were from 2014.

**Comment by geowurster** [comment]
@enriquetuya Yeah we used it in the prototype pipeline, and at one point I think the old data viz repo used it as a git submodule.
